components/draggable/index.js

```diff
@@ -0,0 +1,157 @@
+	 */
+	onDragStart( event ) {
+		const { elementId, transferData, onDragStart = noop } = this.props;
+		const element = document.getElementById( elementId );
+
+		// Set a fake drag image to avoid browser defaults. Remove from DOM right after.
+		// event.dataTransfer.setDragImage is not supported yet in IE,
+		// we need to check for its existance first.
+		if ( 'function' === typeof event.dataTransfer.setDragImage ) {
+			const dragImage = document.createElement( 'div' );
+			dragImage.id = `drag-image-${ elementId }`;
+			dragImage.classList.add( dragImageClass );
+			document.body.appendChild( dragImage );
+			event.dataTransfer.setDragImage( dragImage, 0, 0 );
+			this.props.setTimeout( () => {
+				document.body.removeChild( dragImage );
+			} );
+		}
+
+		event.dataTransfer.setData( 'text', JSON.stringify( transferData ) );
+
+		// Prepare element clone and append to element wrapper.
+		const elementRect = element.getBoundingClientRect();
+		const elementWrapper = element.parentNode;
+		const elementTopOffset = parseInt( elementRect.top, 10 );
+		const elementLeftOffset = parseInt( elementRect.left, 10 );
+		const clone = element.cloneNode( true );
+		clone.id = `clone-${ elementId }`;
+		this.cloneWrapper = document.createElement( 'div' );
+		this.cloneWrapper.classList.add( cloneWrapperClass );
+		this.cloneWrapper.style.width = `${ elementRect.width + ( clonePadding * 2 ) }px`;
+
+		if ( elementRect.height > cloneHeightTransformationBreakpoint ) {
+			// Scale down clone if original element is larger than 700px.
+			this.cloneWrapper.style.transform = 'scale(0.5)';
+			this.cloneWrapper.style.transformOrigin = 'top left';
+			// Position clone near the cursor.
+			this.cloneWrapper.style.top = `${ event.clientY - 100 }px`;
+			this.cloneWrapper.style.left = `${ event.clientX }px`;
+		} else {
+			// Position clone right over the original element (20px padding).
+			this.cloneWrapper.style.top = `${ elementTopOffset - clonePadding }px`;
+			this.cloneWrapper.style.left = `${ elementLeftOffset - clonePadding }px`;
+		}
+
+		// Hack: Remove iFrames as it's causing the embeds drag clone to freeze
+		[ ...clone.querySelectorAll( 'iframe' ) ].forEach( child => child.remove() );
+
+		this.cloneWrapper.appendChild( clone );
+		elementWrapper.appendChild( this.cloneWrapper );
+
+		// Mark the current cursor coordinates.
+		this.cursorLeft = event.clientX;
+		this.cursorTop = event.clientY;
+		// Update cursor to 'grabbing', document wide.
+		document.body.classList.add( 'is-dragging-components-draggable' );
+		document.addEventListener( 'dragover', this.onDragOver );
+
+		this.props.setTimeout( onDragStart );
+	}
+
+	componentWillUnmount() {
+		this.removeDragClone();
+	}
+
+	removeDragClone() {
+		document.removeEventListener( 'dragover', this.onDragOver );
+		if ( this.cloneWrapper && this.cloneWrapper.parentElement ) {
+			// Remove clone.
+			this.cloneWrapper.remove();
```

Comment:
It doesn't look like `.remove` is supported in IE11: https://caniuse.com/#feat=childnode-remove

Elsewhere in the project, `.parentNode.removeChild` is used.